packages/dx-react-grid/docs/reference/data-type-provider.md

```diff
@@ -48,6 +48,10 @@ row? | any | A row.
 value | any | Specifies the editor value.
 onValueChange | (newValue: any) => void | Handles value changes.
 disabled | boolean | **true** if users should not be able to edit the value, **false** otherwise.
```

Comment:
In the majority of our documentation, we use "`otherwise **false**`" instead of "`**false** otherwise`". Can we update this as follows (and in the descriptions below as well)?

```suggestion
disabled | boolean | **true** if users should not be able to edit the value; otherwise **false**.
```